web/pdf_page_view.js

```diff
@@ -1123,17 +1147,7 @@ class PDFPageView {
           await this.#renderAnnotationLayer();
 
           if (this.#enableAutoLinking) {
```

Comment:
Given that you are also checking if `this.#enableAutoLinking` is true inside `#injectLinkAnnotations`, this isn't really needed anymore.

Reply:
While you're obviously correct, the intention was to avoid a "useless" await call when auto-linking is disabled.
However, that might be a pointless micro-optimization?

Reply:
That is probably pointless overall, but I guess "many pointless optimizations" can stack together and have a significant effect across the codebase :)

But then, it's even better to move the whole `this.#enableAutoLinking && this.annotationLayer` check from the method to here.